### PR TITLE
Feature/partial page overhaul

### PR DESCRIPTION
**Pull request by stevenhillcox**

### CODE REVIEW

File: VotingApplication/VotingApplication.Web.Api/Controllers/Web Controllers/AdminController.cs [hunk 1]
```diff
@@ -1,13 +1,13 @@
-﻿using System.Web.Mvc;
+﻿using System;
+using System.Web.Mvc;
 
-namespace VotingApplication.Web.Api.Controllers.Web_Controllers
+namespace VotingApplication.Web.Api.Controllers
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web.Api/Controllers/Web Controllers/AdminController.cs]: If we're changing the namespace, we should change the directory too (as any new items added in will have the namespace derived from the path)

File: VotingApplication/VotingApplication.Web.Api/Scripts/Manage.js [hunk 2]
```diff
@@ -16,6 +16,7 @@
 
                 success: function (data) {
                     self.options(data.Options);
+                    self.pollId(data.UUID);
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web.Api/Scripts/Manage.js]: Should this be manageId, not pollId?

File: VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js [hunk 3]
```diff
@@ -71,56 +76,13 @@
...
-                error: Common.handleError
-            });
-        };
-
-        var votingStrategyFunc = function (strategy, pollData) {
-            function StrategyViewModel() {
-                votingStrategy = new strategy(pollData.Options, pollData);
-
-                //Refresh results every 10 seconds
-                var allResults = function () { self.getResults(self.pollId) };
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js]: We seem to have lost this refreshing

File: VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js [hunk 4]
```diff
@@ -233,14 +193,14 @@
                     Name: username,
                     PollId: pollId,
                     Token: {
-                        TokenGuid: tokenId
+                        TokenGuid: token || Common.sessionItem("token", pollId)
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js]: The top of the file already does this, so we should just need "TokenGuid: token"

File: VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js [hunk 5]
```diff
@@ -292,7 +252,9 @@
                 statusCode: {
                     200: function (data) {
                         lastResultsRequest = Date.now();
-                        votingStrategy.displayResults(data);
+                        if (self.votingStrategy) {
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web.Api/Scripts/Poll.js]: This should include the update to lastResultsRequest too. It doesn't make sense to say we've received the data if we don't display it.

File: VotingApplication/VotingApplication.Web.Api/Scripts/Manage.js (hunk 2 above)
Comment (stevenhillcox) [inline VotingApplication/VotingApplication.Web.Api/Scripts/Manage.js]: No, this is the poll Id used to link the user back to the poll they are managing.

**Pull request merged**